src/parser/monk/mistweaver/modules/spells/InvokeYulon.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Combatants from 'parser/shared/modules/Combatants';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import { formatNumber } from 'common/format';
+import { TooltipElement } from 'common/Tooltip';
+
+const debug = false;
+
+class InvokeYulon extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  petID = null;
+  soothHealing = 0;
+  envelopHealing = 0;
+  healing = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.ENVELOPING_BREATH), this.handleEnvelopingBreath);
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER_PET).spell(SPELLS.SOOTHING_BREATH), this.handleSoothingBreath);
+    this.addEventListener(Events.summon.by(SELECTED_PLAYER), this.handleSummon);
+    debug && this.addEventListener(Events.fightend, this.fightEndDebug);
+  }
+
+  handleSummon(event) {
+    this.petID = event.targetID;
+    debug && console.log(`${event.ability.guid === SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id ? 'Chi-Ji' : 'Yu\'lon'} Summoned: ${this.petID}`);
+  }
+
+  handleEnvelopingBreath(event) {
+    this.envelopHealing += (event.amount || 0) + (event.absorbed || 0);
+  }
+
+  handleSoothingBreath(event) {
+    this.soothHealing += (event.amount || 0) + (event.absorbed || 0);
+  }
+  
+  fightEndDebug() {
+    console.log(`Yu'lon ID: ${this.petID}`);
+  }
+
+  statistic() {
+    if (!this.selectedCombatant.hasTalent(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id)) {
+      return (
+        <>
+          <StatisticBox
+            position={STATISTIC_ORDER.OPTIONAL(50)}
+            icon={<SpellIcon id={SPELLS.SOOTHING_BREATH.id} />}
+            value={`${formatNumber(this.soothHealing)}`}
+            label={(
+              <TooltipElement content="This is the effective healing contributed by Soothing Breath.">
+                Soothing Breath Healing
+              </TooltipElement>
+            )}
+          />
+          <StatisticBox
+            position={STATISTIC_ORDER.OPTIONAL(50)}
+            icon={<SpellIcon id={SPELLS.ENVELOPING_BREATH.id} />}
+            value={`${formatNumber(this.envelopHealing)}`}
+            label={(
+              <TooltipElement content="This is the effective healing contributed by the Enveloping Breath.">
+                Enveloping Breath Healing
+              </TooltipElement>
+            )}
+          />
+          <StatisticBox
```

Comment:
I think 3 statistic boxes is a bit much for one spell. Maybe trim this down to one statistic box with the icon being `summon yulon`, the primary text being total healing, and a tooltip to break it down between enveloping and soothing breath. Or whatever looks best and gives the best info.